### ISSUE TITLE
Running in a docker container, storing data on samba share

### ISSUE THREAD

**Issue by rogierlommers** [containous/traefik#4089]
My situation is like this:

- I run traefik inside a docker container
- I use a locally mounted samba share as the volume where traefik can store its data
- So this means that the acme.json file will also be stored on the samba share
- The problem is dat the samba share is available as `755`, but then traefik gives me the following error: `permissions 755 for /etc/traefik/acme/acme.json are too open, please use 600`

So basically my question is: can I skip the permission checking?

My suggestion would be to create a change which makes this [check](https://github.com/containous/traefik/blob/master/provider/acme/local_store_unix.go#L11) optional. I could send a PR for this. What do you guys think?

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of communication regulation.

Thanks for your interest in Traefik!

We dedicate the issue tracker to bug reports and feature requests only. My advanced AI tells me this one is neither of them.

I encourage your to seek community support: join our [Slack workspace](https://slack.traefik.io/) and reach out to us on the [#support channel](https://traefik.slack.com/messages/C0CDT22PJ/).

Another option is to use [Stack Overflow](https://stackoverflow.com/questions/tagged/traefik) and tag your question `traefik`.

You can of course double check [Traefik's documentation](https://docs.traefik.io/) :sweat_smile:
